### ISSUE TITLE
Tip Should Show in the correct location. 

### ISSUE THREAD

**Issue by Amna-Hassan04** [Amna-Hassan04/Serenity-Guide#95]
![image](https://github.com/user-attachments/assets/dd86c772-e654-4630-a41d-e94895522814)
The tip should show next to the gif but it doesnt. This needs to be fixed. Since this is a tricky and important task I will be giving level 2 for its solution.

**Comment by yasheela-alla** [comment]
@Amna-Hassan04 Do you mean inside the GIF ( Like an overlay ) ?

**Comment by yasheela-alla** [comment]
Hey @Amna-Hassan04, I made few changes to the code, to fix this issue. Please assign me, I will raise a PR

**Comment by Amna-Hassan04** [comment]
@yasheela-alla yes while making PR do add a video recording of the feature  working

**Comment by omkarmakar** [comment]
![image](https://github.com/user-attachments/assets/c1e8bb88-1b13-4396-8aef-867e7ca9614e)
@Amna-Hassan04 does that work fine? or any changes needed visually?

**Comment by zoobiasyed** [comment]
@omkarmakar you did what @Amna-Hassan04 asked for, but can you make sure that the gif is centered like in a way that the girl is in the middle not on left side. Also to mention that the tip and all words should be leveled properly. After the fixes the UI would look more improved version of itself. It's a suggestion if you could work on it it will make it better.
